src/main/java/org/folio/service/mapping/MappingServiceImpl.java

```diff
@@ -53,4 +56,13 @@ private String runDefaultMapping(JsonObject instance, ReferenceData referenceDat
     return this.ruleProcessor.process(entityReader, recordWriter, referenceData, rules);
   }
 
+  @Override
+  public List<VariableField> mapFields(JsonObject record, MappingProfile mappingProfile, String jobExecutionId, OkapiConnectionParams connectionParams) {
+    ReferenceData referenceData = referenceDataProvider.get(jobExecutionId, connectionParams);
+    List<Rule> rules = ruleFactory.create(mappingProfile);
```

Comment:
yea I pulled in the changes from Master now. But one thing I realized is that we are checking for configuration rules presence in the MappingServiceImpl, that should be the job of Rules factory. We should probably move that if we have time

Reply:
I think Rule factory will be in the shared library...Does other modules need functionality to override rules by configs?

Reply:
Good question, But yea I think so , because they override the rules only according to their specific needs so I assume it will be applicable for all exports they are doing, but we can confirm and them move accordingly